.github/workflows/run.yml

```
@@ -33,14 +33,22 @@ jobs:
       shell: bash -el {0}
       run: |
         source /opt/SIRF-SuperBuild/INSTALL/bin/env_sirf.sh
-        curl -fsSL https://raw.githubusercontent.com/SyneRBI/PETRIC/main/petric.py > petric.py
+        dataset=Siemens_mMR_NEMA_IQ
+        curl -sfSL -o  ${dataset}.zip https://petric.tomography.stfc.ac.uk/data/${dataset}.zip
+        mkdir -p data; cd data; unzip -u ../${dataset}.zip; cd ..
         test -f main.py || ln -s main_ISTA.py main.py
         python <<EOF
+        import petric
         from main import Submission, submission_callbacks
         from cil.optimisation.algorithms import Algorithm
         assert issubclass(Submission, Algorithm)
         assert isinstance(submission_callbacks, list)
         assert all(callable(callback) for callback in submission_callbacks)
+        srcdir, outdir, metrics = petric.data_dirs_metrics[0]
+        data = petric.get_data(srcdir)
+        assert data.OSEM_image.max() > 0
+        assert data.reference_image.max() > 0
+        assert data.FOV_mask.max() > 0
```

may as well test init?

```suggestion
        assert data.FOV_mask.max() > 0
        algo = Submission(data)
```

or maybe to avoid e.g. "could not find GPU"...

```suggestion
        assert data.FOV_mask.max() > 0
        try:
            algo = Submission(data)
        except Exception:
            from warnings import warn
            warn("Submission.__init__ failed", RuntimeWarning, stacklevel=2)
```

---

we'd want to have a dummy submission then. Not sure what a test tells us that we know will fail. Anyway, that'll all be for PETRIC 2.0... I suggest to do minor clean-up here, and merge it when all done.